lib/inflections/sv.rb

```diff
@@ -0,0 +1,93 @@
+# encoding: utf-8
+#
+# Swedish inflections
+#
+# Original rules by olabini - https://olabini.com/blog/2006/08/announcing-swedish-rails
+# Adapted by richardjohansson - for https://github.com/mynewsdesk/swedish_pluralize
+#
+module Inflections
+  ActiveSupport::Inflector.inflections(:sv) do |inflect|
+    inflect.clear
+
+    # --- Plural ---
+
+    # -er
+    inflect.plural(/([tmnslpr])$/i, '\1er') # produkt, produkter
+
+    # -or
+    inflect.plural(/a$/i, 'or') # flicka, flickor
+
+    # -ar
+    inflect.plural(/e$/i, 'ar') # pojke, pojkar
+    inflect.plural(/g$/i, 'gar')
+    inflect.plural(/l$/i, 'lar') # bil, bilar
+    inflect.plural(/el$/i, 'lar') # fågel, fåglar
+
+    # -er
+    inflect.plural(/kel$/i, 'kler') # muskel, muskler
+    inflect.plural(/tel$/i, 'tlar') # titel, titlar
+
+    # -r
+    inflect.plural(/de$/i, 'der') # fiende, fiender
+
+    # -n
+    inflect.plural(/le$/i, 'lem') # muskel, muskler
```

Comment:
Well spotted. I don't speak Swedish myself, but that's no excuse for a wrong comment! I took these inflections from directly from here:

https://github.com/mynewsdesk/swedish_pluralize/blob/master/lib/swedish_pluralize.rb

I had to change the order of some of the definitions to get the tests (which I created from the trailing comments) to pass too. I've got no idea if these are correct in fact so I'll get my translator to have a look and get back to you...